kolibri/plugins/learn/assets/src/app.js

```diff
@@ -20,9 +20,11 @@ class LearnModule extends KolibriApp {
     return pluginModule;
   }
   ready() {
-    // If we are not logged in and are forbidden from accessing as guest
-    // redirect to CONTENT_UNAVAILABLE.
     router.beforeEach((to, from, next) => {
+      const redirectPageName = window.sessionStorage.getItem('redirect-after-setup');
```

Comment:
I think this could be more simply done, by just always redirecting to the Library page from `/` when the user is logged in and no content is loaded?